### DISCUSSION TITLE
"greyscale input: true" - how do I turn this off?

### DISCUSSION THREAD

**Discussion by createperhaps** [adieyal/sd-dynamic-prompts#134] [Q&A] [answered]
It is making my images black and white. I never asked this to happen. I don't see a setting to turn it off.

**Comment by adieyal** [comment]
Could you provide a little more information? Where do you see grayscale input: true? Does this still happen if the uncheck the enable dynamic prompts checkbox?

**Reply by createperhaps** [reply]
> Could you provide a little more information? Where do you see grayscale input: true? Does this still happen if the uncheck the enable dynamic prompts checkbox?

It is in the colab window.
"This share link expires in 72 hours. For free permanent hosting and GPU upgrades (NEW!), check out Spaces: https://huggingface.co/spaces
100% 70/70 [00:19<00:00,  3.50it/s]
greyscale input: true

When I disable it, I ran the exact same text2image again, that line goes away, though the particular seed and prompt I re-used to compare is still B&W. I am not sure anymore that it is this plugin making it B&W, but I don't see the line when I disable it.

**Comment by createperhaps** [comment, marked as answer]
I did some more digging.

"INFO:dynamic_prompting.py:Prompt matrix will create 1 images in a total of 1 batches.
100% 70/70 [00:20<00:00,  3.34it/s]
Grayscale input: True"

I get this input when restore face is active, but not when it is off. I don't think it is your plugin anymore.

**Reply by adieyal** [reply]
Marking this as an answer, please open an issue if it happens again and you suspect Dynamic Prompts

**Comment by adieyal** [comment]
Keep me posted.
